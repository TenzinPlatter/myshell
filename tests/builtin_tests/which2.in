which bash echo ls
exit
