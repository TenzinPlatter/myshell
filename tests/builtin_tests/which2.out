>> /usr/sbin/bash
/usr/sbin/echo
/usr/sbin/ls
>> 
